tests/unit/test_intercomm2.c

```diff
@@ -212,8 +198,7 @@ check_file(int iosysid, int format, char *filename, int my_rank, int verbose)
         ERR(ERR_WRONG);
     if ((ret = PIOc_get_att_int(ncid, NC_GLOBAL, ATT_NAME, &att_data)))
         ERR(ret);
-    if (verbose)
-        printf("%d test_intercomm2 att_data = %d\n", my_rank, att_data);
+    printf("%d test_intercomm2 att_data = %d\n", my_rank, att_data);
```

Comment:
Because the build system already provides the feature that tests will run without output. I expected originally that my test print statements would pollute the output with masses of lines of information, so I would have to turn them off with a verbose flag.

But since I don't need to do that, it was always set to 1 and was therefore just useless code.